src/addandremove.js

```diff
@@ -0,0 +1,52 @@
+import { PopulateList } from './liststructure';
+
+function AddList(array) {
+  const input = document.querySelector('#input');
+  const newDescription = input.value;
+  const newTask = {
+    description: newDescription,
+    completed: false,
+    index: array.length,
```

Comment:
* I think the property index be set to the value of the new array length  (i.e. if you're adding a 5th task to the list, the index of that task should equal 5) [Please refer to the project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=By%20default%20new%20tasks%20should%20have%20the%20property%20completed%20set%20to%20false%20and%20the%20property%20index%20set%20to%20the%20value%20of%20the%20new%20array%20length%20(i.e.%20if%20you%27re%20adding%20a%205th%20task%20to%20the%20list%2C%20the%20index%20of%20that%20task%20should%20equal%20to%205).) 
* HINT 💭 You can set your index by doing (index: x.length + 1 ) so that items created don't take the index `0`

Reply:
@davidosky007 thank you, I'm on it.